src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,15 +1,18 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+    private int numberOfTrailer;
+
+    public Excavator(int numberOfTrailer) {
+        this.numberOfTrailer = numberOfTrailer;
+    }
```

Comment:
I think that this constructor should be moved after the default constructor

Reply:
In my opinion, it depends on style code and it is not a big problem.